File: Data Structures & Algorithms/frequency-of-the-most-frequent-element/submission-0.py
class Solution:
    def maxFrequency(self, nums: List[int], k: int) -> int:
        nums.sort()
        res = 1
        for i in range(len(nums)):
            j = i - 1
            tmpK = k
            while j >= 0 and (tmpK - (nums[i] - nums[j])) >= 0:
                tmpK -= (nums[i] - nums[j])
                j -= 1
            
            res = max(res, i - j)
        
        return res

File: Data Structures & Algorithms/max-consecutive-ones-iii/submission-0.java
class Solution {
    public int longestOnes(int[] nums, int k) {
        int res = 0;
        for (int l = 0; l <nums.length; l++){
            int cnt = 0, r = l;

            while (r < nums.length){
                if (nums[r] == 0){
                    if(cnt == k) break;
                    cnt++;
                }
                r++;
            }
            res = Math.max(res, r - l);
        }

        return res;

    }
}